lib/NetworkClientFactory.js

```diff
@@ -1,28 +0,0 @@
-/**
```

Comment:
- will these changes break unit tests?
- did you forget to delete iveDVR/tests/unit/NetworkClientFactory-spec.js as well?
- usages of mockNetwork config var